spring-boot/proxy/src/main/resources/application.yml

```diff
@@ -35,3 +35,13 @@ logging:
   level:
     eu.coatrack.proxy: DEBUG
     org.springframework.security: DEBUG
+
+# The gateway has the ability to work even if the CoatRack admin server is not reachable. If admin is not reachable
+# for longer than this period of time, the gateway will reject all service API requests until a connection could
+# be re-established.
+number-of-minutes-the-gateway-works-without-connection-to-admin: 60
```

Comment:
```suggestion
number-of-minutes-the-gateway-should-work-without-connection-to-admin: 60
```

Reply:
Why are we using "should"? Imo that "should" sounds like we are hoping it will work during this period of time, but we do know that it will work.

Reply:
Feel free to choose a better name, this was just a suggestion to make it more clear that it is a "configuration value" (it defines "what we want to happen") rather than a "statistical/measured value" (what has happend).

Reply:
Good point. I think the words "shall" or "will" would be more suitable.

Reply:
Done.